core/services/workflows/models_yaml.go

```diff
@@ -162,7 +162,7 @@ type stepDefinitionYaml struct {
 	//
 	// Eventually, we might support minor version and specific version pins. This will allow workflow authors to have flexibility when selecting the version, and node operators will be able to determine when they should update their capabilities.
 	//
-	// There are two ways to specify a type - using a string as a fully qualified ID or a structured table. When using a table, tags are ordered alphanumerically and joined into a string following a
+	// There are two ways to specify a type - using a string as a fully qualified ID or a structured table. When using a table, labels are ordered alphanumerically and joined into a string following a
 	//  {type}:{tag1_key}_{tag1_value}:{tag2_key}_{tag2_value}@{version}
```

Comment:
You forgot to edit this line above: {label1_key}_{label1_value} etc

Reply:
Nice catch!